package.json

```diff
@@ -37,7 +37,7 @@
     "eslint": "NODE_PATH=. eslint .",
     "test:nsp": "NODE_PATH=. nsp check",
     "security": "NODE_PATH=. nsp check",
-    "test:functional": "NODE_ENV=test yarn test-contract && NODE_ENV=test ./bin/run-functional-tests.sh",
+    "test:functional": "NODE_ENV=test yarn test-e2e",
```

Comment:
yarn e2e does this, is there a need for this script?
i think we can just remove it if its just running yarn e2e script and replace whereever this is being used.

Reply:
everything else LGTM

Reply:
it's what the jenkins pipeline calls so it needs to exist